vendor.conf

```diff
@@ -2,7 +2,7 @@
 github.com/urfave/cli v1.20.0
 github.com/kr/pretty v0.1.0
 github.com/kr/text v0.1.0
-github.com/containers/image v2.0.0
+github.com/containers/image delete-oci http://github.com/vrothberg/image
```

Comment:
Should there be a WIP titlle to this PR given you're pulling from your private repo?

Reply:
Absolutely, thanks for catching that!